### PR TITLE
LPS-196721 It is possible to make a copy of a not published DPT

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation
[Link to ticket](https://liferay.atlassian.net/browse/LPS-196721)

## Test scenario 1

**Add feature.flag.LPS-195263=true to portal-ext.properties.**

1. Create a Display Page Template named “TestD”
2. View the page in edit mode
3. Add a fragment Button.
4. Don’t publish the page.
5. Navigate to Display Page Templates: Product Menu > Design > Page Templates > Display Page Templates.
6. Click the Kebab Menu of the page you created in step 2.
7. Click on Make a Copy.
8. A new DPT card is displayed with the name “Test D (Copy)”
9. Click on “Test D (Copy)”

### Note: you should also test if you can make a copy in case you have one publish DPT, make some changes and don't publish them (see video attached).

https://github.com/liferay-echo/liferay-portal/assets/93203423/47f0708a-1757-4115-a18d-7b222141fd25

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java [hunk 1]
```diff
@@ -76,6 +76,8 @@ public DisplayPageActionDropdownItemsProvider(
 
 		_draftLayout = LayoutLocalServiceUtil.fetchDraftLayout(
 			layoutPageTemplateEntry.getPlid());
+		_layout = LayoutLocalServiceUtil.fetchLayout(
```
Comment (jkappler) [inline modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java:79]: @BarbaraCabrera, where are we using this variable? Maybe I'm missing something 🤔

Comment (BarbaraCabrera) [reply modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/servlet/taglib/util/DisplayPageActionDropdownItemsProvider.java:79]: My bad, that shouldn't be in the pr.

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
@BarbaraCabrera ❤️

**Comment by jkappler** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**